### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by viniciusccarvalho**
sh -> bash
docker_host -> docker_server
Adding wait for scdf server

### CODE REVIEW

File: scripts/local/common.sh [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
+#!/bin/bash
 
 
 # ======================================= FUNCTIONS START =======================================
 
 function stop_docker_container() {
 
-  [ -z "$DOCKER_HOST" ] && { echo "Environment variable DOCKER_HOST must be set"; exit 1; }
-  DOCKER_SERVER=$(echo $DOCKER_HOST | awk -F/ '{print $3}' | awk -F':' '{print $1}')
+  [ -z "$DOCKER_SERVER" ] && { echo "Environment variable DOCKER_HOST must be set"; exit 1; }
```
Comment (trisberg) [inline scripts/local/common.sh]: should this say "Environment variable DOCKER_SERVER must be set"?

File: scripts/cloudfoundry/init/env.properties [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SPACE=metrics
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_API_TIMEOUT=300
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_URL=
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SKIP_SSL_VALIDATION=true
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_USERNAME=
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_PASSWORD=
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_STREAM_SERVICES=rabbit
 SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_DOMAIN=
-SPRING_CLOUD_DATAFLOW_SERVER_DOWNLOAD_URL=http://repo.spring.io/snapshot/org/springframework/cloud/spring-cloud-dataflow-server-cloudfoundry/1.2.0.BUILD-SNAPSHOT/spring-cloud-dataflow-server-cloudfoundry-1.2.0.BUILD-SNAPSHOT.jar
+SPRING_CLOUD_DATAFLOW_SERVER_DOWNLOAD_URL=http://repo.spring.io/libs-snapshot/org/springframework/cloud/spring-cloud-dataflow-server-cloudfoundry/1.1.1.BUILD-SNAPSHOT/spring-cloud-dataflow-server-cloudfoundry-1.1.1.BUILD-20170215.152655-32.jar
```
Comment (trisberg) [inline scripts/cloudfoundry/init/env.properties]: why using an older release?

**Comment by trisberg** [comment]
Should this now pull `1.2.1.BUILD-SNAPSHOT` jars?

File: scripts/cloudfoundry/init/env.properties (hunk 2 above)
Comment (viniciusccarvalho) [inline scripts/cloudfoundry/init/env.properties]: I'll bump this back to 1.2.x release. This file is actually a placeholder that users of the tests are expected to override most entries.

**Comment by trisberg** [comment]
LGTM, merging

**Comment by trisberg** [comment]
Squashed and merged as commit d2a59f27bb997133ba19388d49c791e0196205ba

**Pull request closed without merging**